include/deal.II/multigrid/mg_coarse.h

```diff
@@ -351,6 +432,90 @@ ::set_matrix(const MatrixType &m)
   matrix = new PointerMatrix<MatrixType, VectorType>(&m);
 }
 
+// -------
+
+template <class VectorType,
+          class SolverType,
+          class MatrixType,
+          class PreconditionerType
+          >
+MGCoarseGridIterativeSolver<VectorType, SolverType, MatrixType, PreconditionerType>
+::MGCoarseGridIterativeSolver ()
+  :
+  solver(0, typeid(*this).name()),
+  matrix(0, typeid(*this).name()),
+  preconditioner(0, typeid(*this).name())
+{}
+
```

Comment:
it's both 2 and 3 lines within this file. Probably that should be documented somewhere, i guess `astyle` does not pick this up.